diagnostic.md

```diff
@@ -32,23 +40,24 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+Getter methods: .val(), .find()
+Setter methods: .val(), .html()
```

Comment:
for these to be setters they need to be passed an argument like a string to set the value to.